src/server/pbsd_main.c

```diff
@@ -1450,6 +1450,11 @@ main(int argc, char **argv)
 		/* in case secondary didn't remove the file */
 		/* also tells the secondary to go idle	    */
 		(void)unlink(path_secondaryact);
+		/* 
+		 * Make the scheduler (re)-read the configuration
+		 * and fairshare usage.
+		 */
+		(void)contact_sched(SCH_CONFIGURE, NULL, pbs_scheduler_addr, pbs_scheduler_port); 
```

Comment:
Don't we need to check if contact_sched is indeed successful here? What if contact_sched fails and at this point of time if scheduler is down.

Reply:
Also here we are sending SCH_CONFIGURE to re-read the configuration and fairshare usage. But EDD says we send SCH_SCHEDULE_FIRST. Does EDD needs an update or Am I missing anything ?

Reply:
@suresh-thelkar - I have updated the EDD.
about checking if contact_sched being successful here, I feel there is no need.
Scheduler is started by the script or verified by the init script to be already running.
This part of the code is for primary, and we do not have failover for scheduler process.

Reply:
@prakashcv13,
Thanks for updating the EDD. On not checking the return value of contact_sched I agree with you if we start primary server always with the init script. If this is the case I feel it is better to mention this assumption also in the EDD.  But I don't think this this is the case always for example consider the case of Windows and also another case of while the primary is up , stop it using qterm and after a while bring it up using the script pbs_server alone.